gui/public/i18n/ko/translation.ftl

```diff
@@ -397,10 +396,10 @@ onboarding-wifi_creds-skip = Wi-Fi 설정 건너뛰기
 onboarding-wifi_creds-submit = 저장!
 onboarding-wifi_creds-ssid =
     .label = SSID
```

Comment:
Can you change SSID to Wifi name?

Reply:
Not a big deal, but would it be easier for users to understand?

Reply:
We are forcing the change so we don't use a technical name like SSID